enclave/core/CMakeLists.txt

```diff
@@ -208,7 +208,7 @@ if (OE_SGX)
     target_compile_options(oecore PUBLIC
         -fPIE
         -nostdinc
-        -fno-stack-protector
+        -fstack-protector-strong
```

Comment:
Are we fine pushing this onto consumers of the SDK?

Reply:
Thanks, it is removed.